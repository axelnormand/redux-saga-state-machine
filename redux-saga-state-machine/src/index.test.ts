import { createStateMachineSaga } from './index';

describe('createStateMachineSaga', () => {
  it('returns a generator function', () => {
    const description = {};
    const result = createStateMachineSaga(description);
    expect(result).toEqual();
  });
});
