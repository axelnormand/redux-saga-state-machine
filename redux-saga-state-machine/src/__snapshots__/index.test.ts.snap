// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createStateMachineSaga returns a generator function 1`] = `
Object {
  "next": [Function],
  "return": [Function],
  "throw": [Function],
  Symbol(Symbol.iterator): [Function],
}
`;
